src/libs/actions/Report.ts

```diff
@@ -1139,7 +1139,7 @@ function expandURLPreview(reportID: string, reportActionID: string) {
 }
 
 /** Marks the new report actions as read */
-function readNewestAction(reportID: string) {
+function readNewestAction(reportID: string, shouldResetUnread = false) {
```

Comment:
`shouldResetUnreadMarker` is a better name, IMO
And can we add a comment here to explain the parameter

Reply:
@tienifr Friendly bump, wdyt about the naming suggestion ^ ?